packages/aws-cdk-lib/aws-s3-deployment/README.md

```diff
@@ -199,6 +199,41 @@ new s3deploy.BucketDeployment(this, 'DeployMeWithoutDeletingFilesOnDestination',
 });
 ```
 
+### Explicit Prune Validation
+
+To prevent accidental data loss, you can enable the feature flag `@aws-cdk/aws-s3-deployment:requireExplicitPrune` 
```

Comment:
we should be clear about exactly what scenario "accidental data loss" happens. and also link to feature flag documentation